### PR TITLE
Updated Solidity docs for building from source and binaries

### PR DESCRIPTION
**Pull request by bobsummerwill**
They now reflect our current status, and refer to the 'install_deps.sh' script.

### CODE REVIEW

File: docs/installing-solidity.rst [hunk 1]
```diff
@@ -29,105 +29,78 @@ To install it, simply use
...
 `solc-js repository <https://github.com/ethereum/solc-js>`_.
 
-Binary Packages
+Binary Releases
 ===============
 
-Binary packages of Solidity together with its IDE Mix are available through
-the `C++ bundle <https://github.com/ethereum/webthree-umbrella/releases>`_ of
-Ethereum.
+The `solidity-v0.3.6 <https://github.com/ethereum/solidity/releases/tag/v0.3.6>`_
```
Comment (chriseth) [inline docs/installing-solidity.rst:35]: Is this paragraph really relevant for he documentation?

File: docs/installing-solidity.rst [hunk 2]
```diff
@@ -29,105 +29,78 @@ To install it, simply use
...
 
 
-Requirements:
+Building from Source (macOS, Debian or Ubuntu)
+==============================================
 
-- OS X Yosemite (10.10.5)
-- Homebrew
-- Xcode
+As of August 2016 we now support an **install_deps.sh** script which encodes all of the
```
Comment (chriseth) [inline docs/installing-solidity.rst:57]: Is the date really relevant?

Comment (bobsummerwill) [inline docs/installing-solidity.rst:57]: Yes, until we fix the binaries, this detailed context is important.
We're being asked repeatedly.
We can strip it back down again when we have the binaries back.

File: docs/installing-solidity.rst [hunk 3]
```diff
@@ -29,105 +29,78 @@ To install it, simply use
...
-    sudo apt-get -y update
-    sudo apt-get -y upgrade # this will update cmake to version 3.x
-    sudo apt-get -y install libcryptopp-dev libjsoncpp-dev
+Building from Source (Windows)
+======================================
 
-Ubuntu Xenial (16.04)
----------------------
+Pre-requisites
+--------------------------------------------------------------------------------
```
Comment (Denton-L) [inline docs/installing-solidity.rst:76]: Can you make the line the same length as the title above?

File: docs/installing-solidity.rst [hunk 4]
```diff
@@ -29,105 +29,78 @@ To install it, simply use
...
-    sudo apt-get -y install build-essential git cmake libgmp-dev libboost-all-dev \
-        libjsoncpp-dev
-    
-    sudo add-apt-repository -y ppa:ethereum/ethereum
-    sudo add-apt-repository -y ppa:ethereum/ethereum-dev
-    sudo apt-get -y update
-    sudo apt-get -y upgrade # this will update cmake to version 3.x
-    sudo apt-get -y install libcryptopp-dev libjsoncpp-dev
+Building from Source (Windows)
+======================================
```
Comment (Denton-L) [inline docs/installing-solidity.rst:73]: Also, make this line the same length as the above.

**Comment by bobsummerwill** [comment]
Superceded by https://github.com/ethereum/solidity/pull/946.

**Pull request closed without merging**